nrcatalogtools/utils.py

```diff
@@ -1,10 +1,10 @@
 import functools
 import pathlib
 import shutil
-
 import lal
 import requests
 import sxs
+from multiprocessing import Process, Queue
```

Comment:
Pls move this import back inside the function `call_with_timeout`. Currently `multiprocessing` is not used anywhere else, and `call_with_timeout` is not used anywhere , so this import will be an unnecessary burden when importing `utils`..